index.js

```diff
@@ -1,18 +1,133 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
+//share files within project
+const lib = require('./ask');
+
+//choosing which game to play first
+gameChoice();
+async function gameChoice() {
+  let userGameChoice = await lib.ask('Do you want to guess first or should I? \nPress "1" if you want to guess first and press "2" if you want me to guess first. \n')
+  if (parseInt(userGameChoice) === 1) {
+    startHumGuessNum();
+  } else if (parseInt(userGameChoice) === 2) {
+    startCompGuessNum();
+  } else {
+    console.log("Well that's not a choice at all! I guess you don't want to play any of my games :( Goodbye!");
+    process.exit();
+  };
```

Comment:
Harsh error handling!  All joking aside, good job checking for errors.  If you wanted to just let them try to type again though you could encapsulate the choice in a small loop.

Reply:
Haha ok I was pretty sick of guess the number when I wrote the first version, but you'll see on the next commit that I have more patience for the game now that I've stepped away from it and I'm letting people try again :)